aidial_adapter_openai/gpt.py

```diff
@@ -14,7 +21,20 @@
     generate_stream,
     map_stream,
 )
-from aidial_adapter_openai.utils.tokens import Tokenizer, truncate_prompt
+from aidial_adapter_openai.utils.tokens import Tokenizer
+
+
+def plain_message_truncate(
```

Comment:
Maybe `text_messages_truncate` to elaborate on what `plain` actually mean? Versus `multimodal_message_truncate`

Reply:
fixed